modules/deepspace/assets/ships/alienAttacker/alienattacker.json

```diff
@@ -0,0 +1,78 @@
+/*
+Ship sprite from MillionthVector is licensed under a Creative Commons Attribution 4.0 International License.
+millionthvector.blogspot.id
+*/
+
+
+{
+  "size": 0.6,
```

Comment:
Use spaces of size 4, instead of 2. (that's the size that all other json files - and your files - use.)